### ISSUE TITLE
Any plan to support windows?

### ISSUE THREAD

**Issue by spindensity** [rcarriga/vim-ultest#71]
This plugin works very well on linux, but does not work on windows, since the `pty` module does not work on windows according to python's official documentation(in fact, it does not work on windows at all since there's no `termios` module):
https://github.com/rcarriga/vim-ultest/blob/dfea06dc0e8da24338eef3414259055f360449cb/rplugin/python3/ultest/handler/runner/handle.py#L1-L5

> The pty module defines operations for handling the pseudo-terminal concept: starting another process and being able to write to and read from its controlling terminal programmatically.
>
> Because pseudo-terminal handling is highly platform dependent, there is code to do it only for Linux. (The Linux code is supposed to work on other platforms, but hasn’t been tested yet.)

Any plan to support on windows?

**Comment by rcarriga** [comment]
Yep I've thought about this before but I do not develop on windows and do not have the time to get setup and be able to test. I'd certainly be interested in adding support for it though, I believe the only issue would be in the `handle.py` file, which can be altered though it would mean that the `UltestAttach` command would not be available as it requires the pty module. If there is an alternative solution I'd be open to hearing it, but for first steps I'd be happy with just getting the rest working. 

I don't have much time right now to work on this but I'd be open to PRs, otherwise I'll get around to it when I have a chance :smile:

**Comment by DasOhmoff** [comment]
Yeah, please support Windows, that would be great! :)

**Comment by ViktorHefringMarine** [comment]
Same here, would love to be able to use this plugin on Windows.

**Comment by rcarriga** [comment]
If you are using NeoVim I'd recommend checking out my new plugin https://github.com/rcarriga/neotest which is a successor to vim-ultest. It *should* support windows (if not let me know in an issue please) as it's not using any Unix specific code
